plugins/woocommerce-admin/client/wp-admin-scripts/product-tracking/shared.ts

```diff
@@ -324,4 +324,23 @@ export const initProductScreenTracks = () => {
 				} );
 			}
 		} );
+
+	document
+		.querySelector(
+			'#woocommerce-product-updated-message-view-product__link'
+		)
+		?.addEventListener( 'click', () => {
+			recordEvent( 'product_view_product_click', getProductData() );
```

Comment:
`getProductData()` is not returning all the properties we need to, to match `wcadmin_product_update`. See acceptance criteria on #35494.

You'll want to update `getProductData()` to include the same fields as in `wcadmin_product_update`.

See https://github.com/woocommerce/woocommerce/blob/19058ed17c63f5cae9b49d0694ebff8db0dccc42/plugins/woocommerce/includes/tracks/events/class-wc-products-tracking.php#L164